.concourse/pipeline.yaml.gomplate

```diff
@@ -727,6 +728,102 @@ jobs:
 {{- end }}
 {{ $previousTest = (printf "cf-acceptance-tests-%s-%s" $cfScheduler $branch) }}
 
+- name: cats-internetless-{{ $cfScheduler }}-{{ $branch }}
+  public: true
+  plan:
+  - get: kubecf-{{ $branch }}
+    passed:
+    - {{ $previousTest | quote }}
+    trigger: true
+    version: "every"
+  - get: s3.kubecf-ci
+    passed:
+    - {{ $previousTest | quote }}
+  - get: s3.kubecf-ci-bundle
+    passed:
+    - {{ $previousTest | quote }}
+  - get: semver.gke-cluster
+    passed:
+    - {{ $previousTest | quote }}
+    trigger: true
+  - get: catapult
+{{- if has $stable (printf "cats-internetless-tests-%s" $cfScheduler) }}
+  - put: status-{{ $branch }}.src
+    params: &cats_internetless_{{ $cfScheduler }}_{{ $sanitized_branch_name }}_status
+        context: "cats-internetless-tests-{{ $cfScheduler }}"
+        description: "Internetless acceptance tests {{ $cfScheduler }}"
+        path: kubecf-{{ $branch }}/{{ $path }}
+        state: pending
+{{- end }}
+  - task: test-{{ $cfScheduler }}
+    timeout: 5h30m
+    input_mapping:
+      kubecf: kubecf-{{ $branch }}
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: splatform/catapult
+      inputs:
+      - name: catapult
+      - name: kubecf
+      - name: semver.gke-cluster
+      params:
+        DEFAULT_STACK: cflinuxfs3
```

Comment:
`        DEFAULT_STACK: cflinuxfs3` Is this needed anymore?

Reply:
You are right, it's not used. Removing.